znai-java/src/main/java/com/twosigma/znai/java/extensions/JavaIncludePlugin.java

```diff
@@ -51,14 +56,15 @@ public IncludePlugin create() {
     public JavaIncludeResult process(JavaCode javaCode) {
         opts = pluginParams.getOpts();
 
-        Boolean bodyOnly = opts.get("bodyOnly", false);
-        Boolean signatureOnly = opts.get("signatureOnly", false);
+        isBodyOnly = opts.get("bodyOnly", false);
+        isSignatureOnly = opts.get("signatureOnly", false);
+        isMultipleEntries = !entries.isEmpty();
```

Comment:
Should this be isMultipleEntries = entries.size() > 1? It appears that way based on the name and how it's used below (line 100).